GUI/add_new.py

```diff
@@ -30,7 +32,7 @@ def check_url(url):
 
 def game_exists(game_name, db=None):
 	if db is None:
-		with open('games.json', 'r') as jsonfile:
+		with urlopen('http://dogeek.legtux.org/games.json') as jsonfile:
```

Comment:
Why use an external site instead of github for hosting the json? In my experience using external sites tends to be a bad idea because the moment someone leaves the project/forgets to renew the domain/gets hacked/gets an SSL cert not trusted by all major OSs an update has to pushed out. Not to mention that any fork of this project would have to rewrite this.

Reply:
Apparently, there's a library called gitpython that allows you to interact with repositories, though it needs to have git installed, which will be an issue on windows machines.